quick-start.md

```diff
@@ -7,55 +7,82 @@ description: Get an instance of TinyMCE up and running in less than 5 minutes.
 
-With the script included, initialize {{site.productname}} {{site.productmajorversion}} on any element (or elements) in the web page.
+```html
+<script>
+  tinymce.init({
+    selector: '#mytextarea'
+  });
+</script>
+```
 
-Since {{site.productname}} enables identifying replaceable elements via a CSS selector, the only requirement is to pass an object that contains a `selector` to `tinymce.init()`.
+**For example**:
 
-In this example, replace `<textarea id='mytextarea'>` with a {{site.productname}} {{site.productmajorversion}} editor instance by passing the selector `'#mytextarea'` to `tinymce.init()`.
+To replace `<textarea id='mytextarea'>` with a {{site.productname}} {{site.productmajorversion}} editor instance, pass the selector `'#mytextarea'` to `tinymce.init()`.
 
 ```html
 <!DOCTYPE html>
 <html>
-<head>
-  <script src='{{ site.cdnurl }}' referrerpolicy="origin"></script>
-  <script>
-  tinymce.init({
-    selector: '#mytextarea'
-  });
-  </script>
-</head>
-
-<body>
-<h1>{{site.productname}} Quick Start Guide</h1>
-  <form method="post">
-    <textarea id="mytextarea" name="mytextarea">Hello, World!</textarea>
-  </form>
-</body>
+  <head>
+    <script src='{{ site.cdnurl }}' referrerpolicy="origin"></script>
+    <script>
+      tinymce.init({
+        selector: '#mytextarea'
+      });
+    </script>
+  </head>
+
+  <body>
+  <h1>{{site.productname}} Quick Start Guide</h1>
+    <form method="post">
+      <textarea id="mytextarea">Hello, World!</textarea>
+    </form>
+  </body>
 </html>
 ```
 
+Adding this content to a HTML file and opening it in a web browser will load a TinyMCE editor, such as:
+
+{% include codepen.html id="default" %}
+
+To remove the notice:
+
+<img alt="warning icon" src="{{ site.baseurl }}/images/icons/warning.svg" style="vertical-align:middle;"/>**_This domain is not registered with {{site.cloudname}}..._**
+
+Replace `no-api-key` in the source script (`<script src=...`) with a [{{site.cloudname}} API key]({{site.accountsignup}}).
+
+Signing up for a {{site.cloudname}} API key will also provide a trial of the [Premium Plugins]({{site.baseurl}}/enterprise/).
 
 ## Step 3: Saving content with a form POST
+Process the content with a form handler.
+
+When the `<form>` is submitted, the {{site.productname}} {{site.productmajorversion}} editor mimics the behavior of a normal HTML `<textarea>` during the `POST`. The host's form handler can process the submitted content in the same way as content from a regular `<textarea>`.
 
-When the `<form>` is submitted, the {{site.productname}} {{site.productmajorversion}} editor mimics the behavior of a normal HTML `<textarea>` during the `POST`. In the user's form handler, the content submitted can be processed in the same way as the content created from a regular `<textarea>`.
+The HTML elements and inline CSS are also submitted, along with pasted media such as images. Images are submitted as inline content with base64 encoding.
```

Comment:
This isn't quite true for the base64 comment, it's only for images that haven't been uploaded, so that could maybe be confusing?

Reply:
fair. should I delete the last sentence?

Reply:
It might be easier than trying to explain it in a quick start, but I'm not sure honestly.

Reply:
removed

Reply:
I... find this whole sentence quite confusing tbh. Could we try a different angle like "All editor content is submitted including all HTML elements, inline CSS and images."? Although I think it's "submitted" here that's really bugging me since that seems... technically incorrect?